src/scripts/policy/apply_blueprint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+# shellcheck disable=SC1083,SC1090,SC1091,2154
+# SC1083: This is literal.
+# SC1090: Can't follow non-constant source. Use a directive to specify location.
+# SC1091: Not following. Shellcheck can't follow non-constant source.
+# SC2154: "var is referenced but not assigned". These values come from an external file.
+#
+# This script will apply and publish a local blueprint as well as apply it to the target subscription
+
+set -e
+
+error_log() {
+  echo "${1}" 1>&2;
+}
+
+usage() {
+  echo "apply_blueprint.sh: Deploy a locally stored blueprint to a subscription"
+  error_log "usage: apply_blueprint.sh <subscription> <location> <blueprint_name>"
+}
+
+if [[ "$#" -lt 2 ]]; then
+    usage
+    exit 1
+fi
+
+subscription_id=$1
+location=$2
+blueprint_name=$3
+
+timestamp=$(date +%s)
+
+# Blueprint requires an extension, lets set it to install that
+az config set extension.use_dynamic_install=yes_without_prompt
+
+# For execution we need some locations
+blueprints=$(realpath ../policy/blueprints/)
+scripts_path=$(realpath ../)
+
+thisblueprint="${blueprints}/${blueprint_name}"
+blueprint_parameters="${blueprints}/parameters/${blueprint_name}.json"
+
+# Validate template exists
+. "${scripts_path}/util/checkforfile.sh" \
+   "${thisblueprint}" \
+   "The blueprint definition name could not be found, please check spelling and that you have copied the template to the src/scripts/policy/blueprints directory"
```

Comment:
I can't seem to pass the right argument. It is the directory name that I'm supposed to pass, right?

Couldn't get these to pass the `checkforfile` validation:

```plaintext
./apply_blueprint.sh {mysubid} eastus blueprintsample
./apply_blueprint.sh {mysubid} eastus Blueprint.json
./apply_blueprint.sh {mysubid} eastus blueprintsample/Blueprint.json
```

Reply:
Ah, I take it the argument is supposed to be a directory and azext_blueprint looks for a file named `blueprint.json` in the given directory, but the default we have in the repo is named `Blueprint.json`

Reply:
Did some reading, and I'm assuming this is because I'm on a case-sensitive file system. In any event, looks like the az CLI blueprint extension should support export (including the lower named `blueprint.json`) soon:

https://github.com/Azure/azure-cli-extensions/commit/372b3f2c4c0a015986b91c0e489cda91998e0265#diff-5d5c99398a53f9720ff0625ee86d343eb01259d1dfd1c14a48b9668cabd26403R129-R133